Altis_Life.Altis/core/items/fn_boltcutter.sqf

```diff
@@ -6,48 +6,68 @@
     Description:
     Breaks the lock on a single door (Closet door to the player).
 */
-private ["_building","_door","_doors","_cpRate","_title","_progressBar","_titleText","_cp","_ui"];
-_building = param [0,objNull,[objNull]];
+private ["_building", "_door", "_doors", "_cpRate", "_title", "_progressBar", "_titleText", "_cp", "_ui"];
+_building = param [0, objNull, [objNull]];
+private _home = false;
+private _ownerName = "";
 
 private _vaultHouse = [[["Altis", "Land_Research_house_V1_F"], ["Tanoa", "Land_Medevac_house_V1_F"]]] call TON_fnc_terrainSort;
-private _altisArray = [16019.5,16952.9,0];
-private _tanoaArray = [11074.2,11501.5,0.00137329];
+private _altisArray = [16019.5, 16952.9, 0];
+private _tanoaArray = [11074.2, 11501.5, 0.00137329];
 private _pos = [[["Altis", _altisArray], ["Tanoa", _tanoaArray]]] call TON_fnc_terrainSort;
 
 if (isNull _building) exitWith {};
 if (!(_building isKindOf "House_F")) exitWith {hint localize "STR_ISTR_Bolt_NotNear";};
-if (((nearestObject [_pos,"Land_Dome_Big_F"]) == _building || (nearestObject [_pos,_vaultHouse]) == _building) && (west countSide playableUnits < (LIFE_SETTINGS(getNumber,"minimum_cops")))) exitWith {
-    hint format [localize "STR_Civ_NotEnoughCops",(LIFE_SETTINGS(getNumber,"minimum_cops"))];
+if (((nearestObject [_pos, "Land_Dome_Big_F"]) == _building || (nearestObject [_pos, _vaultHouse]) == _building) && (west countSide playableUnits < (LIFE_SETTINGS(getNumber, "minimum_cops")))) exitWith {
```

Comment:
Could [`isEqualTo`](https://community.bistudio.com/wiki/isEqualTo) be used instead of [`==`](https://community.bistudio.com/wiki/a%20==%20b)? 

```sqf
if (((nearestObject [_pos, "Land_Dome_Big_F"]) isEqualTo _building || (nearestObject [_pos, _vaultHouse]) isEqualTo _building) && (west countSide playableUnits < (LIFE_SETTINGS(getNumber, "minimum_cops")))) exitWith {
```